index.html

```diff
@@ -0,0 +1,968 @@
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+          <div class="mobile">
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- section 3 -->
+      <section class="alignment">
+        <div class="title flex block">
+          <div id="div5">
+            <h1>Latest funding rounds on Index.co</h1>
+            <i class="fas fa-angle-right"></i>
+          </div>
+          <div>
+            <a class="red" href="#">Add info |</a
+            ><a class="red" href="#"> Get updates by email |</a
+            ><a class="red" href="#"> See more</a>
+          </div>
+        </div>
+
+        <div class="grid section3">
+          <div class="majordiv">
+            <h1>Amwell</h1>
+            <div class="div1">
+              <p>$100M</p>
+              <p class="grey">CORPORATE ROUND</p>
+            </div>
+            <div class="div2">
```

Comment:
#### Mobile Screen Size

- [ ] The text on the right side of the image should not be centralized. Should be closer to the image.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91436839-47003100-e83f-11ea-8309-9713463c09da.png) |![image](https://user-images.githubusercontent.com/61479648/91436871-554e4d00-e83f-11ea-9cd4-afffa3a34532.png) |